homeassistant/components/homeassistant/triggers/state.py

```diff
@@ -30,7 +30,7 @@
 BASE_SCHEMA = cv.TRIGGER_BASE_SCHEMA.extend(
     {
         vol.Required(CONF_PLATFORM): "state",
-        vol.Required(CONF_ENTITY_ID): cv.entity_ids,
+        vol.Required(CONF_ENTITY_ID): cv.template_or_entity_ids,
```

Comment:
I don't think we should add this logic, as it can be problematic in terms of how it works / how one excepts this to work.

The problem is, is that templates are dynamic. The results of them change over time, however, that is not the case for this trigger. If the result of the template changes, the trigger doesn't adjust or change with it.